how-it-works/3_governance/01_network-nervous-system-nns.subpage.md

```diff
@@ -5,27 +5,91 @@ shareImage: /img/how-it-works/network-nervous-system.webp
+A proposal is adopted if either of the two conditions are met:
+1) the voting period has elapsed, there are more votes in favor than against the proposal (simple majority), and at least 3% of the totally available voting power voted in favor of the proposal (enough participation).
+2) more than 50% of the total voting power are in favor of the proposal. At this point, the proposal's result cannot be changed anymore even if all remaining voters vote against the proposal and therefore the proposal is executed without awaiting the end of the voting period.
+
+
+## Proposal execution
+As soon as a proposal is adopted, it is executed by the NNS. With very few exceptions, proposals are executed automatically and fully on-chain.
+
+
+
+
+## What the NNS governs
+The NNS governs the full Internet Computer blockchain.
+For example, the NNS makes decisions on the following topics.
+* *Network topology*: The NNS decides which node machines can join the network and how these nodes are organized into _subnets_.
+* *Software run on the nodes*: The NNS decides which software is run by the nodes that form the blockchain. This allows the NNS to upgrade the protocol without manual intervention of the individual node operators.
+* *Tokenomics*: The NNS decides how many rewards node operators and governance participants get and the costs of computation and storage on the Internet Computer.
+* *Governance*: The NNS decides on the rules of the governance itself, e.g. how many votes are needed for a proposal to be adopted, how long the voting period is, etc.
+
+
+All these decisions are executed fully on-chain and at low cost compared to other chains. Therefore, decisions can be made efficiently and frequently allowing the NNS to adapt the ICP platform to the community's needs.
+
+
+## The Architecture of the NNS
+The NNS is implemented as a set of _canister smart contracts_ on a dedicated _NNS system subnet_.
+Facilitated by the low computation and storage costs on the ICP, and since the NNS subnet is cross-subsidized by other subnets, voting on proposals is free of charge and a high volume of votes and proposals can be processed and stored on-chain.
+
+
+Two important canisters are the _governance canister_ that manages neurons, proposals, and the voting process, and the _ICP ledger canister_ that implements the ICP token which is the NNS governance token.
+In addition, there are other canisters for special purposes, for example the _cycles minting canister (CMC)_ that is responsible to mint _cycles_ - the "gas" used for computation and storage on the ICP.
+
+
+To allow for flexible governance evolution, the NNS canisters are _mutable smart contracts_.
+Two canisters, called _root_ (written in Rust) and _lifeline_ (written in Motoko), are responsible for upgrading the NNS canisters. As NNS canisters control each other, it is ensured that they can only be changed by an NNS DAO decision.
```

Comment:
Is it relevant to mention the coding language here ?

Reply:
Removed.